### DISCUSSION TITLE
Sidebar Expand and Non Expand

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#850] [Ideas]
### What feature would you like to see?

I think if the user has set his Look & Feel to > Expand Tabs on Hover (Wont work on compact mode). Then the Expanded sidebar should not have influence on the website UI. Like the website where user is need is not change if the Tabs are on hover or not.

![2024-08-21 21_27_12-Window](https://github.com/user-attachments/assets/17a4c2a6-7ecb-486d-b57b-b0446e4bed76)
![2024-08-21 21_27_26-Window](https://github.com/user-attachments/assets/dd61a1db-091d-4790-8461-3a2012dd041f)

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/390_

**Comment by matdave** [comment]
I am not sure why this bug was turned into a discussion.
